Mathlib/RingTheory/HahnSeries/PowerSeries.lean

```diff
@@ -0,0 +1,249 @@
+/-
+Copyright (c) 2021 Aaron Anderson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Aaron Anderson
+-/
+import Mathlib.RingTheory.HahnSeries.Multiplication
+import Mathlib.RingTheory.PowerSeries.Basic
+import Mathlib.Data.Finsupp.PWO
+
+#align_import ring_theory.hahn_series from "leanprover-community/mathlib"@"a484a7d0eade4e1268f4fb402859b6686037f965"
+
+/-!
+# Comparison between Hahn series and power series
+If `Γ` is ordered and `R` has zero, then `HahnSeries Γ R` consists of formal series over `Γ` with
+coefficients in `R`, whose supports are partially well-ordered. With further structure on `R` and
+`Γ`, we can add further structure on `HahnSeries Γ R`.  When `R` is a semiring and `Γ = ℕ`, then
+we get the more familiar semiring of formal power series with coefficients in `R`.
+## Main Definitions
+  * `toPowerSeries` the isomorphism from `HahnSeries ℕ R` to `PowerSeries R`.
+  * `ofPowerSeries` the inverse, casting a `PowerSeries R` to a `HahnSeries ℕ R`.
```

Comment:
This also appears in the `PowerSeries.lean` file. I guess that's where it belongs?